integration/indexes_compat_test.go

```diff
@@ -565,17 +576,17 @@ func TestIndexesDropRunCommand(t *testing.T) {
 	t.Parallel()
 
 	for name, tc := range map[string]struct { //nolint:vet // for readability
```

Comment:
This will create a lot of conflict, see the DoD of https://github.com/FerretDB/FerretDB/issues/2599

Reply:
And I've just moved them in this PR https://github.com/FerretDB/FerretDB/pull/2806

Reply:
@chilagrow let's merge your PR first, yes